packages/apps/docs/.prettierignore

```diff
@@ -0,0 +1,2 @@
+.next
+**/*.md
```

Comment:
why igone md files?

Reply:
Because we have a separate formatter for Markdown files and also because we have quite some generated Markdown files (e.g. from API Extractor).